src/player/components/PlayerCard/PlayerCard.tsx

```diff
@@ -0,0 +1,105 @@
+import { useCallback, useMemo } from 'react';
+import classNames from 'clsx';
+
+import type { Character, PlayerCardProps } from '@player/models';
+import Button from '@shared/Button/components/Button';
+import { COLORS } from '@core/constants';
+import Icon from '@shared/Icon/components/Icon';
+import Image from '@shared/Image/components/Image';
+import { isLoggedPlayer } from '@player/helpers';
+import PlayerLevel from '@player/components/PlayerLevel';
+import useClientSideValue from '@hooks/useClientSideValue';
+import useTranslation from '@hooks/useTranslation';
+
+/**
+ * PlayerCard component.
+ *
+ * @param {PlayerCardProps} props Props
+ * @return {JSX.Element}
+ */
+export const PlayerCard = ({
+  className = '',
+  direction = 'right',
+  hideLevel = false,
+  hideName = false,
+  isLoading = false,
+  onEdit,
+  player,
+  priority,
+  size = 'large',
+}: PlayerCardProps): JSX.Element => {
+  const { translate } = useTranslation();
+
+  const name = useClientSideValue(!isLoading ? player?.pseudo : translate('waiting'));
+
+  const renderCharacterImage = useCallback(() => {
+    // TODO: replace by character api data
+    const character: Character =
+      !!player && isLoggedPlayer(player)
```

Comment:
Sauf erreur de ma part
```suggestion
      player && isLoggedPlayer(player)
```